color/quoted.h

```diff
@@ -29,16 +29,40 @@
 #include "core/lib.h"
 #include "color.h"
 
+struct Buffer;
+
 /// Ten colours, quoted0..quoted9 (quoted and quoted0 are equivalent)
 #define COLOR_QUOTES_MAX 10
 
 extern int QuotedColors[];
 extern int NumQuotedColors;
 
+/**
+ * struct QClass - Style of quoted text
+ *
+ * NeoMutt will store a tree of all the different email quoting levels it
+ * detects in an Email.  If $quote_regex matches, say both "> " and "| ",
+ * and the Email has three levels of indent, then the tree will contain two
+ * siblings each with a child and grandchild.
+ */
+struct QClass
```

Comment:
As a reader, `QClass` doesn't much context to the struct's function. Any objections to change it to `Quote` or something along those lines?

Reply:
Changed to `QuoteStyle`